docs/source/_static/html_snippets/chapters_content_directory_tree_only_yaml.html

```diff
@@ -1,11 +1,11 @@
 <div id="tree-diagram">
-	├── <span class="directory">algorithms</span><br>
-	│&nbsp;&nbsp;&nbsp;├── <span class="file">algorithms.yaml</span><br>
-	│&nbsp;&nbsp;&nbsp;└── <span class="directory">sections</span><br>
-	│&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;└── <span class="file">sections.yaml</span><br> class="file">lessons.yaml</span><br>
-	├── <span class="directory">artificial-intelligence</span><br>
-	│&nbsp;&nbsp;&nbsp;├── <span class="file">artificial-intelligence.yaml</span><br>
-	│&nbsp;&nbsp;&nbsp;└── <span class="directory">sections</span><br>
-	│&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;└── <span class="file">sections.yaml</span><br>
+	├── <span class="directory">algorithms/</span><br>
+	│   ├── <span class="file">algorithms.yaml/</span><br>
+	│   ├── <span class="directory">sections/</span><br>
+	│   └──	└── <span class="file">sections.yaml/</span><br>
```

Comment:
I assume this line should be `│            └──` Spaced appropriately

Same a couple lines down

Reply:
Not quite sure what you mean here sorry! What should be there instead?

Reply:
```
│   ├── <span class="directory">sections/</span><br>
│   └──	└── <span class="file">sections.yaml/</span><br>
```
Should be:
```
│   └── <span class="directory">sections/</span><br>
│      	└── <span class="file">sections.yaml/</span><br>
```
OR:
```
│   ├── <span class="directory">sections/</span><br>
│   └── <span class="file">sections.yaml/</span><br>
```
You gotta make the lines line up properly

Reply:
ok sweet will do thanks!